odoo/addons/base/static/src/scss/res_partner.scss

```diff
@@ -6,3 +6,7 @@
         --gutter-x: 0;
     }
 }
+
+.o_kanban_partner_footer_buttons a:first-child{
```

Comment:
I always prefer when specific styling is at least included in a main class of the res_partner whole form.

Otherwise it's a bit lost.